### DISCUSSION TITLE
Live stream on Kick platform

### DISCUSSION THREAD

**Discussion by vtxx1** [ant-media/Ant-Media-Server#5678] [FAQs]
Hello, has anyone managed to do live on the Kick platform? I'm using the community version, but when I enter the transmission key provided by Kick, my live is still offline.

**Comment by USAMAWIZARD** [comment]
I have never used kick , but let me check and get back to you

**Comment by USAMAWIZARD** [comment]
@vtxx1 yes you are right there seems to be some problem while streaming to kick , let me create an issues on github.

**Reply by mekya** [reply]
Here is the GitHub issue link to follow for your convenience.
https://github.com/ant-media/Ant-Media-Server/issues/5683
